ansible/roles/dropbear_initramfs/tasks/main.yml

```diff
@@ -53,7 +53,7 @@
     user: '{{ item.user | d("root") }}'
     path: '{{ "/etc/initramfs-tools/root/.ssh/authorized_keys"
               if ("dropbear" in dropbear_initramfs__base_packages)
-              else "/etc/dropbear-initramfs/authorized_keys" }}'
+              else "{{ dropbear_initramfs__config_path }}/authorized_keys" }}'
```

Comment:
You are already in Jinja `{{ }}` here :)

Reply:
Thx for this advice! I fixed it with: https://github.com/debops/debops/pull/2479/commits/1ead1242222080b50ec0799be7cf0b04e7ce1b5b